CHANGELOG.md

```diff
@@ -6,20 +6,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### New feature support
 #### Cisco Resources
-- `cisco_aaa_authentication_login` type and provider.
-- `cisco_aaa_authorization_login_cfg_svc` type and provider.
-- `cisco_aaa_authorization_login_exec_svc` type and provider.
-- `cisco_aaa_group_tacacs` type and provider.
-- `cisco_acl` type and provider
-- `cisco_evpn_vni` type and provider.
-- `cisco_vni` type and provider.
 - `cisco_vxlan_global` type and provider.
-- `cisco_vxlan_vtep` type and provider.
+- `cisco_ace` type and provider.
```

Comment:
Also note that all of the entries in this file (and the README) need to be kept in alpha order.